### PR TITLE
d/aws_batch_job_definition 

### LINKED ISSUE #28665: [New Data Source]: aws_batch_job_definition
### Description

I have a project that creates resources in AWS Batch (job queues, job definitions, compute environments), and a separate project that consumes those resources. While I understand that I can rely on Terraform Remote State to convey information from one project to another, in this case I'd like to avoid that and would prefer to use discrete data lookups in the downstream project.

Since Batch Job Definitions support multiple revisions, I would presume by default the data source should retrieve the latest revision.

### Requested Resource(s) and/or Data Source(s)

* data.aws_batch_job_definition

### Potential Terraform Configuration

```terraform
data "aws_batch_job_definition" "main" {
  name = "my-job-definition"
}

data "aws_batch_job_queue" "main" {
  name = "my-job-queue"
}

locals {
  job_def_arn_less_revision = regex("^(.*)(:\\d+)$", data.aws_batch_job_definition.main.arn)
}

data "aws_iam_policy_document" "batch_execute" {
  statement {
    sid       = "AllowBatchExecute"
    actions   = ["batch:SubmitJob"]
    effect    = "allow"
    resources = [
      local.job_def_arn_less_revision,
      data.aws_batch_job_queue.main.arn
    ]
  }
}
```

### References

_No response_

### Would you like to implement a fix?

None

### PR DESCRIPTION
**Pull request by danquack**
### Description
Creates a data source for `batch_job_definition`.

### Relations
closes #28665

### Output from Acceptance Testing

```console
% make testacc TESTS="TestAccBatchJobDefinitionDataSource" PKG=batch                        
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/batch/... -v -count 1 -parallel 20 -run='TestAccBatchJobDefinitionDataSource'  -timeout 360m
=== RUN   TestAccBatchJobDefinitionDataSource_basicName
=== PAUSE TestAccBatchJobDefinitionDataSource_basicName
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
=== CONT  TestAccBatchJobDefinitionDataSource_basicName
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
--- PASS: TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties (16.03s)
--- PASS: TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties (17.46s)
--- PASS: TestAccBatchJobDefinitionDataSource_basicName (26.61s)
--- PASS: TestAccBatchJobDefinitionDataSource_basicARN (26.78s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/batch	28.724s
```

### CODE REVIEW

File: internal/service/batch/job_definition_data_source.go [hunk 1]
```diff
@@ -0,0 +1,1341 @@
...
+}
+
+func (d *dataSourceJobDefinition) Metadata(_ context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) { // nosemgrep:ci.meta-in-func-name
+	resp.TypeName = "aws_batch_job_definition"
+}
+
+func (d *dataSourceJobDefinition) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"arn": schema.StringAttribute{
```
Comment (drewmullen) [inline internal/service/batch/job_definition_data_source.go:61]: Can you add an `arn_prefix` attribute please

**Review by drewmullen — CHANGES_REQUESTED** [review]
1 requested change

**Review by drewmullen — APPROVED** [review]
LGTM!

```shell
make testacc TESTS="TestAccBatchJobDefinitionDataSource" PKG=batch  
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/batch/... -v -count 1 -parallel 20 -run='TestAccBatchJobDefinitionDataSource'  -timeout 360m
=== RUN   TestAccBatchJobDefinitionDataSource_basicName
=== PAUSE TestAccBatchJobDefinitionDataSource_basicName
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
=== CONT  TestAccBatchJobDefinitionDataSource_basicName
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
--- PASS: TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties (41.02s)
--- PASS: TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties (42.70s)
--- PASS: TestAccBatchJobDefinitionDataSource_basicName (57.54s)
--- PASS: TestAccBatchJobDefinitionDataSource_basicARN (67.12s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/batch      69.494s
```

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

```console
% make testacc TESTARGS='-run=TestAccBatchJobDefinitionDataSource_' PKG=batch ACCTEST_PARALLELISM=2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/batch/... -v -count 1 -parallel 2  -run=TestAccBatchJobDefinitionDataSource_ -timeout 360m
=== RUN   TestAccBatchJobDefinitionDataSource_basicName
=== PAUSE TestAccBatchJobDefinitionDataSource_basicName
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
=== CONT  TestAccBatchJobDefinitionDataSource_basicName
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
--- PASS: TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties (17.97s)
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
--- PASS: TestAccBatchJobDefinitionDataSource_basicName (30.52s)
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN
--- PASS: TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties (16.76s)
--- PASS: TestAccBatchJobDefinitionDataSource_basicARN (26.84s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/batch	68.476s
```

**Comment by ewbankkit** [comment]
@danquack Thanks for the contribution :tada: :clap:.

**Comment by danquack** [comment]
I noticed you switched the data source from using Single Object -> List to work with autoflex. By switching to lists, should these tests also work without an index? One of the reasons I did object originally was for the purity of not having to do `.0`, but that may be something we want to do with the provider (#35813).

https://github.com/hashicorp/terraform-provider-aws/blob/4a89becdea4f21ee3985ef3e61499becb741e3da/internal/service/batch/job_definition_data_source_test.go#L132-L133

Also, when I run the test command I am seeing some panics in the auto flexing. I see yours passed, but maybe it is something in my local development setup, just borked?

```shell
make testacc TESTARGS='-run=TestAccBatchJobDefinitionDataSource_' PKG=batch ACCTEST_PARALLELISM=2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/batch/... -v -count 1 -parallel 2  -run=TestAccBatchJobDefinitionDataSource_ -timeout 360m
=== RUN   TestAccBatchJobDefinitionDataSource_basicName
=== PAUSE TestAccBatchJobDefinitionDataSource_basicName
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== RUN   TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
=== PAUSE TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
=== CONT  TestAccBatchJobDefinitionDataSource_basicName
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
=== NAME  TestAccBatchJobDefinitionDataSource_basicName
    job_definition_data_source_test.go:23: Step 1/2 error: Check failed: Check 3/3 error: data.aws_batch_job_definition.test: Attribute 'retry_strategy.attempts' not found
=== NAME  TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties
    job_definition_data_source_test.go:91: Step 1/1 error: Check failed: Check 1/3 error: data.aws_batch_job_definition.test: Attribute 'node_properties.main_node' not found
--- FAIL: TestAccBatchJobDefinitionDataSource_basicName (10.53s)
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN_EKSProperties
--- FAIL: TestAccBatchJobDefinitionDataSource_basicARN_NodeProperties (10.59s)
=== CONT  TestAccBatchJobDefinitionDataSource_basicARN
panic: Value Conversion Error

An unexpected error was encountered trying to convert from struct value. This is always an error in the provider. Please report the following to the provider developer:

error retrieving field names from struct tags: [0].limits: can't use field name for both Limits and Requests
[0]

goroutine 10330 [running]:
github.com/hashicorp/terraform-provider-aws/internal/errs.Must[...](...)
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/errs/must.go:13
github.com/hashicorp/terraform-provider-aws/internal/errs/fwdiag.Must[...](...)
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/errs/fwdiag/must.go:17
github.com/hashicorp/terraform-provider-aws/internal/framework/types.newListNestedObjectValueOf[...]({0x113262c30, 0x14004633860}, {0x110558960, 0x14002d68de0})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/types/list_nested_objectof.go:236 +0x138
github.com/hashicorp/terraform-provider-aws/internal/framework/types.NewListNestedObjectValueOfSlice[...](...)
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/types/list_nested_objectof.go:228
github.com/hashicorp/terraform-provider-aws/internal/framework/types.NewListNestedObjectValueOfPtr[...](...)
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/types/list_nested_objectof.go:224
github.com/hashicorp/terraform-provider-aws/internal/framework/types.listNestedObjectTypeOf[...].ValueFromObjectPtr(0x1132e1280?, {0x113262c30?, 0x14004633860}, {0x110510600, 0x14002d4b180})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/types/list_nested_objectof.go:115 +0xd4
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.structToNestedObject({}, {0x113262c30, 0x14004633860}, {0x11234ea20?, 0x14002d68b10?, 0x140036247e8?}, 0xc4?, {0x1295abcd8, 0x14004f47930}, {0x112d84860?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:721 +0x22c
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.struct_({}, {0x113262c30, 0x14004633860}, {0x11234ea20?, 0x14002d68b10?, 0x14004f47920?}, 0x68?, {0x113278ad8?, 0x14004f47930?}, {0x112d84860?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:304 +0x20c
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.ptr({}, {0x113262c30, 0x14004633860}, {0x1111436e0?, 0x1400b3b5ad0?, 0x1132d85e8?}, {0x113278ad8?, 0x14004f47930?}, {0x112d84860?, 0x1400b5f9bd8?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:284 +0x3d4
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.convert({}, {0x113262c30, 0x14004633860}, {0x1111436e0?, 0x1400b3b5ad0?, 0x1400acae780?}, {0x112d84860?, 0x1400b5f9bd8?, 0x102ee53a0?})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:76 +0x250
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlexConvertStruct({0x113262c30, 0x14004633860}, {0x112d8f700?, 0x1400b3b5a70?}, {0x1105106c0?, 0x1400b5f9b00?}, {0x1130dea00, 0x11aff1940})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/autoflex.go:110 +0x2c4
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.sliceOfStructNestedObjectCollection({}, {0x113262c30, 0x14004633860}, {0x1105614e0?, 0x1400d950f00?, 0x1121355c0?}, {0x1295abba8, 0x14004f478c0}, {0x112d84c80?, 0x14002cf4480?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:762 +0x42c
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.slice({}, {0x113262c30, 0x14004633860}, {0x1105614e0?, 0x1400d950f00?, 0x1132d85e8?}, {0x113278778?, 0x14004f478c0}, {0x112d84c80?, 0x14002cf4480?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:489 +0x450
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.convert({}, {0x113262c30, 0x14004633860}, {0x1105614e0?, 0x1400d950f00?, 0x1400acae648?}, {0x112d84c80?, 0x14002cf4480?, 0x102ee53a0?})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:80 +0x2a4
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlexConvertStruct({0x113262c30, 0x14004633860}, {0x112bd4820?, 0x1400d950f00?}, {0x110510840?, 0x14002cf4480?}, {0x1130dea00, 0x11aff1940})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/autoflex.go:110 +0x2c4
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.structToNestedObject({}, {0x113262c30, 0x14004633860}, {0x112bd4820?, 0x1400d950cc0?, 0x111b68800?}, 0x0?, {0x1295abb48, 0x14004f47830}, {0x112d854c0?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:715 +0x190
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.struct_({}, {0x113262c30, 0x14004633860}, {0x112bd4820?, 0x1400d950cc0?, 0x112d85360?}, 0xa9?, {0x1132780b8?, 0x14004f47830?}, {0x112d854c0?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:304 +0x20c
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.ptr({}, {0x113262c30, 0x14004633860}, {0x111143b00?, 0x14004f47730?, 0x1132d85e8?}, {0x1132780b8?, 0x14004f47830?}, {0x112d854c0?, 0x14006ee6de0?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:284 +0x3d4
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.convert({}, {0x113262c30, 0x14004633860}, {0x111143b00?, 0x14004f47730?, 0x1400acae510?}, {0x112d854c0?, 0x14006ee6de0?, 0x102ee53a0?})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:76 +0x250
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlexConvertStruct({0x113262c30, 0x14004633860}, {0x11206bf20?, 0x14004f47730?}, {0x110510880?, 0x14006ee6de0?}, {0x1130dea00, 0x11aff1940})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/autoflex.go:110 +0x2c4
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.structToNestedObject({}, {0x113262c30, 0x14004633860}, {0x11206bf20?, 0x14004f47570?, 0x14004f47701?}, 0x28?, {0x129479e90, 0x14004f47720}, {0x112d85620?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:715 +0x190
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.struct_({}, {0x113262c30, 0x14004633860}, {0x11206bf20?, 0x14004f47570?, 0x111b68800?}, 0x40?, {0x113277f98?, 0x14004f47720?}, {0x112d85620?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:304 +0x20c
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.ptr({}, {0x113262c30, 0x14004633860}, {0x111143c20?, 0x140072b2df8?, 0x0?}, {0x113277f98?, 0x14004f47720?}, {0x112d85620?, 0x14007644830?, ...})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:284 +0x3d4
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlattener.convert({}, {0x113262c30, 0x14004633860}, {0x111143c20?, 0x140072b2df8?, 0x1400acae3c8?}, {0x112d85620?, 0x14007644830?, 0x102ee5300?})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:76 +0x250
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlexConvertStruct({0x113262c30, 0x14004633860}, {0x111144160?, 0x140072b2dc0?}, {0x1105108c0?, 0x14007644800?}, {0x1130dea20, 0x11aff1940})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/autoflex.go:110 +0x2c4
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.autoFlexConvert({0x113262c30, 0x14004633860}, {0x111144160, 0x140072b2dc0}, {0x1105108c0, 0x14007644800}, {0x1130dea20?, 0x11aff1940?})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/autoflex.go:48 +0x1c8
github.com/hashicorp/terraform-provider-aws/internal/framework/flex.Flatten({0x113262c30, 0x14004633860}, {0x111144160, 0x140072b2dc0}, {0x1105108c0, 0x14007644800}, {0x0, 0x0, 0x1105108c0?})
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/framework/flex/auto_flatten.go:36 +0xb0
github.com/hashicorp/terraform-provider-aws/internal/service/batch.(*jobDefinitionDataSource).Read(0x14006d9fc30, {0x113262c30, 0x14004633860}, {{{{0x113292ea0, 0x14006fbb740}, {0x11138c440, 0x140041ff6b0}}, {0x1132a21d8, 0x14005a372c0}}, {{{0x0, ...}, ...}, ...}}, ...)
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/service/batch/job_definition_data_source.go:193 +0x5cc
github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider.(*wrappedDataSource).Read(0x14002d4a800, {0x113262c30?, 0x1400508b800?}, {{{{0x113292ea0, 0x14006fbb740}, {0x11138c440, 0x140041ff6b0}}, {0x1132a21d8, 0x14005a372c0}}, {{{0x0, ...}, ...}, ...}}, ...)
	/Users/danielquackenbush/code/aws/terraform-provider-aws/internal/provider/fwprovider/intercept.go:173 +0x100
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ReadDataSource(0x14005190680, {0x113262c30, 0x1400508b800}, 0x1400508b830, 0x14003627608)
	/Users/danielquackenbush/go/pkg/mod/github.com/hashicorp/terraform-plugin-framework@v1.5.0/internal/fwserver/server_readdatasource.go:79 +0x394
github.com/hashicorp/terraform-plugin-framework/internal/proto5server.(*Server).ReadDataSource(0x14005190680, {0x113262c30?, 0x1400508b560?}, 0x140084d9240)
	/Users/danielquackenbush/go/pkg/mod/github.com/hashicorp/terraform-plugin-framework@v1.5.0/internal/proto5server/server_readdatasource.go:55 +0x31c
github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ReadDataSource(0x113262c68?, {0x113262c30?, 0x1400508b290?}, 0x140084d9240)
	/Users/danielquackenbush/go/pkg/mod/github.com/hashicorp/terraform-plugin-mux@v0.14.0/tf5muxserver/mux_server_ReadDataSource.go:36 +0x18c
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadDataSource(0x1400b7d41e0, {0x113262c30?, 0x1400508aab0?}, 0x14001ff41e0)
	/Users/danielquackenbush/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.21.0/tfprotov5/tf5server/server.go:685 +0x344
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadDataSource_Handler({0x112e280e0?, 0x1400b7d41e0}, {0x113262c30, 0x1400508aab0}, 0x1400538f300, 0x0)
	/Users/danielquackenbush/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.21.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:572 +0x164
google.golang.org/grpc.(*Server).processUnaryRPC(0x1400652c000, {0x113262c30, 0x1400508aa20}, {0x113295740, 0x1400269b380}, 0x14002cf9680, 0x1400731ddd0, 0x11af99660, 0x0)
	/Users/danielquackenbush/go/pkg/mod/google.golang.org/grpc@v1.61.0/server.go:1385 +0xb8c
google.golang.org/grpc.(*Server).handleStream(0x1400652c000, {0x113295740, 0x1400269b380}, 0x14002cf9680)
	/Users/danielquackenbush/go/pkg/mod/google.golang.org/grpc@v1.61.0/server.go:1796 +0xc44
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	/Users/danielquackenbush/go/pkg/mod/google.golang.org/grpc@v1.61.0/server.go:1029 +0x8c
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 9254
	/Users/danielquackenbush/go/pkg/mod/google.golang.org/grpc@v1.61.0/server.go:1040 +0x150
FAIL	github.com/hashicorp/terraform-provider-aws/internal/service/batch	20.710s
FAIL
make: *** [testacc] Error 1
```

**Comment by ewbankkit** [comment]
Actually, something seems to have gone awry with that last merge. They were working...

**Comment by ewbankkit** [comment]
374458ac696367a1556a85022fde13b014b3273a fixes those final problems.

**Pull request merged**